main.js

```diff
@@ -0,0 +1,164 @@
+const searchBar = document.querySelector("input");
+const searchButton = document.querySelector(".btn");
+const currentWeatherDiv = document.querySelector(".currentWeather");
+const weather_api_key = "5dfb3ccdb6e27979cf32863254a75835";
+
+const kelvinToFahrenheit = function (number) {
+  let convertedNum = ((number - 273.15) * 9) / 5 + 32;
+  return convertedNum.toFixed(1);
+};
+
+const formatDate = function (utc) {
+  let utcSeconds = utc;
+  let day = new Date(0);
+  day.setUTCSeconds(utcSeconds);
+  return day.toString().slice(0, 3);
+};
+
+const formatDayOfWeek = function (day) {
+  if (day === "Mon") {
```

Comment:
This would be cleaner as a switch https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch